tensorflow/contrib/rnn/python/ops/rnn_cell.py

```diff
@@ -1427,50 +1416,27 @@ def build(self, inputs_shape):
       "candidate_linear_h/kernel",
       shape=[self._num_units, self._num_units])
 
-    self._update_gate_bias = self.add_variable(
-      "update_gate/bias",
-      shape=[self._num_units]) if not self._layer_norm else None
-    self._reset_gate_bias = self.add_variable(
-      "reset_gate/bias",
-      shape=[self._num_units]) if not self._layer_norm else None
-    self._candidate_linear_x_bias = self.add_variable(
-      "candidate_linear_x/bias",
-      shape=[self._num_units]) if not self._layer_norm else None
-    self._candidate_linear_h_bias = self.add_variable(
-      "candidate_linear_h/bias",
-      shape=[self._num_units]) if not self._layer_norm else None
-
     self.built = True
 
   def call(self, inputs, state):
     """GRU cell with layer normalization."""
 
     args = array_ops.concat([inputs, state], 1)
 
-    z = self._linear(args,
-                     kernel=self._update_gate_kernel,
-                     bias=self._update_gate_bias)
-    r = self._linear(args,
-                     kernel=self._reset_gate_kernel,
-                     bias=self._reset_gate_bias)
+    z = math_ops.matmul(args, self._update_gate_kernel)
```

Comment:
just wanted to make sure that you are no longer adding a bias on purpose?

Reply:
Yes, when using layer normalization we should not use bias.